streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -116,4 +121,13 @@ static void executeAndMaybeSwallow(final boolean clean,
             }
         }
     }
+
+    @Override
+    public ProcessorTopology getProcessorTopology() {
```

Comment:
Why do we need this helper function? Shouldn't it be protected?

Reply:
Idea was that mutable variable should be 'protected' with method calls from external places. But agree `protected` would work better here.